WordPress/src/main/java/org/wordpress/android/ui/stats/refresh/lists/sections/viewholders/BarChartViewHolder.kt

```diff
@@ -42,6 +46,24 @@ class BarChartViewHolder(parent: ViewGroup) : BlockListItemViewHolder(
         GlobalScope.launch(Dispatchers.Main) {
             delay(50)
             chart.draw(item, labelStart, labelEnd)
+            chart.post {
+                accessibilityHelper = BarChartAccessibilityHelper(
+                        chart,
+                        contentDescriptions = item.entryContentDescriptions,
+                        hasOverlappingEntries = item.overlappingEntries != null
+                )
+
+                accessibilityHelper.accessibilityEvent = object : BarChartAccessibilityEvent {
+                    override fun onHighlight(index: Int, hasOverlappingEntries: Boolean) {
+                        chart.highlightColumn(index, hasOverlappingEntries)
+                        val entry = chart.data.dataSets.first().getEntryForIndex(index)
```

Comment:
Yes, I should add a comment indeed. I called first here because the chart will always have at least one dataset that does the default rendering i.e the values that the chart is trying to represent. Based on my exploration, it seems three datasets can get applied: main bars being represented, overlapping bars, and highlights and since the main set of values are always added first it will always be the first dataset present. 

Eg. 
main values = Views
overlapping values = Visitors
highlights = selected bar(s) 

I didn't see a unique identifier I could identify each dataset which would make the solution more direct so I opted for this approach.